courses/intro-to-spark/book/02_Flow_Analysis.rst

```diff
@@ -818,12 +837,12 @@ Example #5
 ~~~~~~~~~~
 
 Permutations are modeled as arrays where the element at index ``I`` is the
-position of the ``I``th element in the permutation. The procedure
-``Init`` initializes a permutation to be the identity, the ``I``th
-elements is at the ``I``th position. ``Cyclic_Permuation`` calls ``Init``
-and then swaps the elements until it has constructed a cyclic permutation.
+position of the ``I`` th element in the permutation. The procedure ``Init``
+initializes a permutation to be the identity, where the ``I`` th elements is at
+the ``I`` th position. ``Cyclic_Permuation`` calls ``Init`` and then swaps the
```

Comment:
Typo (already in the original revision): `Cyclic_Permuation` => `Cyclic_Permutation`

Reply:
fixed